components/tyk-bootstrap/values.yaml

```diff
@@ -0,0 +1,124 @@
+# Default values for tyk-bootstrap.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
+  license:
+    # The license key needed for the Tyk Dashboard.
+    dashboard: ""
+
+  servicePorts:
+    # The port at which the dashboard service can be found at
+    dashboard: 3000
+    # The port at which the gateway service can be found at
+    gateway: 8080
+
+  components:
+    # Enables/disables the dashboard component
+    dashboard: false
```

Comment:
what does this field refer to in bootstrap values? If it is enabled, will the bootstrap create a dashboard environment? If so, why is it `false` by default?

Reply:
can you please update the comment accordingly?